src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -139,6 +139,34 @@ const render = (Component: React.ComponentType) => {
   }
 };
 
+function changeFavicon(theme: 'dark' | 'light'): void {
+  const n = document.querySelector<HTMLLinkElement>('.js-site-favicon[type="image/png"]');
+  if (!n) {
+    return;
+  }
+
+  const path = n.href.split('/sentry/')[0];
+  if (theme === 'dark') {
+    n.href = `${path}/sentry/images/favicon-dark.png`;
+  } else {
+    n.href = `${path}/sentry/images/favicon.png`;
```

Comment:
Is this needed?

Reply:
The browser could transition to light mode from dark mode during the day, this would switch the favicon back.